docs/src/appendix/sncast/account/deploy.md

```diff
@@ -13,4 +13,24 @@ Name of the (previously created) account to be deployed.
 ## `--max-fee, -m <MAX_FEE>`
 Optional.
 
-Maximum fee for the `deploy_account` transaction. When not used, defaults to auto-estimation.
+Maximum fee for the `deploy_account` transaction in Fri or Wei depending on fee token or transaction version. When not used, defaults to auto-estimation.
+
+## `--fee-token <FEE_TOKEN>`
+Optional. Required if `--version` is not provided.
+
+Token used for fee payment. Possible values: ETH, STRK.
+
+## `--max-gas <MAX_GAS>`
+Optional.
+
+Maximum gas for the `deploy_account` transaction. When not used, defaults to auto-estimation. (Only for STRK fee payment)
+
+## ` --max-gas-unit-price <MAX_GAS_UNIT_PRICE>`
+Optional.
+
+Maximum gas unit price for the `deploy_account` transaction paid in Fri. When not used, defaults to auto-estimation. (Only for STRK fee payment)
+
+## `--version, -v <VERSION>`
+Optional. Required if `--fee-token` is not provided.
+
+Version of the account deployment transaction. Possible values: v1, v3.
```

Comment:
Wondering if possible values should be `v3` or `3`. `--version` or `-v` flag is self-explanatory imo.

Reply:
I would prefer to leave it as it is. In  [starknet docs](https://docs.starknet.io/architecture-and-concepts/network-architecture/transactions/) they always refer to this as `v3` or `v1`